docs/UserGuide.md

```diff
@@ -300,33 +300,35 @@ by adding your first Delivery
 
 2. After registering for an account, you should be greeted by some sample Customers
    that have been generated by HomeBoss.
-3. Let's assume that a new customer, Gabriel, is to be added, using the `customer add`{.swift} command.
 
-> `NAME`: Gabriel
+3. Let's assume that a new Customer, Alex, is to be added, using the `customer add`{.swift} command.
+
+> `NAME`: Alex
 > `PHONE`: 87654321
-> `EMAIL`: gabrielrocks@gmail.com
+> `EMAIL`: alexrocks@gmail.com
 > `ADDRESS`: RVRC Block B
 
-4. Type `customer add --name Gabriel --phone 87654321 --email gabrielrocks@gmail.com --address RVRC Block B`{.swift}
+4. Type `customer add --name Alex --phone 87654321 --email alexrocks@gmail.com --address RVRC Block B`{.swift}
    into the Command Box and hit enter.
-5. A new Customer called Gabriel should be added, you should see a new Customer similar to the image below.
+
+5. A new Customer called Alex should be added, you should see a new Customer similar to the image below.
 
 ![CustomerAdded](images/getting-started/customer_add.png){style="display: block; margin: 0 auto;"}
 
-6. Now, let's add a new delivery for Gabriel into HomeBoss, suppose Gabriel ordered some furniture.
+6. Now, let's add a new Delivery for Alex into HomeBoss, suppose Alex ordered some furniture.
 
-> `DELIVERY_NAME`: Macbook Pro
+> `DELIVERY_NAME`: furniture
 > `DELIVERY_DATE`: 2023-12-03
```

Comment:
Same as above, the values were specifically chosen because of the screenshot

Reply:
kk